### PR TITLE
Main cookiefactoryv2 staging

### PR DESCRIPTION
**Pull request by intpix**
Added sites template file and updated README.

### CODE REVIEW

File: src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/config/sites.ts [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-  {
-    awsConfig: {
-      sceneId: 'CookieFactory',
-      workspaceId: WORKSPACE_ID
-    },
-    id: crypto.randomUUID(),
-    location: '1 Main Street, Bakersville, NC, USA',
-    name: 'Bakersville Central'
-  },
-  // {
```
Comment (johnnyw-aws) [inline src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/config/sites.ts:17]: oh, I don't remember whether I removed this locally and forgot to revert... I think having multiple sites is fine if you want to bring it back

File: src/workspaces/cookiefactoryv2/README.md [hunk 2]
```diff
@@ -162,7 +193,7 @@ For any issue not addressed here, please open an issue or contact AWS Support.
    sh cloud9\_resize.sh 20
    df -h  
    ```
-### No space during `cdk deploy`: `OSError: [Errno 28] No space left on device` 
+### No space during `cdk deploy: OSError: [Errno 28] No space left on device`
```
Comment (johnnyw-aws) [inline src/workspaces/cookiefactoryv2/README.md:196]: fyi that the 2 backtick sections was intentional here, the former was the command and the latter was the error message, but I don't feel strongly if you feel merging into 1 block is more readable

**Review by johnnyw-aws — APPROVED** [review]

File: src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/config/sites.ts (hunk 1 above)
Comment (intpix) [reply src/workspaces/cookiefactoryv2/CookieFactoryDemo/src/config/sites.ts:17]: I just removed this file in favor of a `sites.template.ts`. made a not about it in the README update.

File: src/workspaces/cookiefactoryv2/README.md (hunk 2 above)
Comment (intpix) [reply src/workspaces/cookiefactoryv2/README.md:196]: ah, sorry. i must've changed accidentally.

**Pull request merged**